web_src/js/components/DashboardRepoList.js

```diff
@@ -0,0 +1,370 @@
+import Vue from 'vue';
+import {initVueSvg, vueDelimiters} from './VueComponentLoader.js';
+
+const {AppSubUrl, AssetUrlPrefix, PageData} = window.config;
+
+function initVueComponents() {
+  Vue.component('repo-search', {
+    delimiters: vueDelimiters,
+    props: {
+      searchLimit: {
+        type: Number,
+        default: 10
+      },
+      subUrl: {
+        type: String,
+        required: true
+      },
+      uid: {
+        type: Number,
+        default: 0
+      },
+      teamId: {
+        type: Number,
+        required: false,
+        default: 0
+      },
+      organizations: {
+        type: Array,
+        default: () => [],
+      },
+      isOrganization: {
+        type: Boolean,
+        default: true
```

Comment:
```suggestion
        default: false
```
Why would you assume by default that our user is an organization?

Reply:
As above, no big change to first copy-move refactor.